pom.xml

```diff
@@ -1682,11 +1693,57 @@
         <surefire.reuseForks>${reuseForks}</surefire.reuseForks>
       </properties>
     </profile>
+    <profile>
+      <!--
+        Explicitly add the log4j annotation processor to generate the Log4jPlugins.dat file for the
+        AccumuloMonitorAppender, but only when not using the errorprone profile. This is necessary
+        because log4j added a GraalVMProcessor that we don't need and won't run without failing the
+        build, so we need to itemize all the processors we do want to run. Except, I couldn't get
+        errorprone to work when I tried to itemize its processors, so errorprone needs to rely on
+        the normal autodiscovery process. Since that would pick up the unwanted GraalVMProcessor,
+        this profile exists to avoid adding log4j-core to the processors when using errorprone.
+        This prevents using the AccumuloMonitorAppender when using the errorprone profile.
+        Activate the errorprone profile and de-activate this one simultaneously by using:
+        `mvn -Derrorprone` or `mvn -Perrorprone,!generate-log4j-plugin-data`
+      -->
+      <id>generate-log4j-plugin-data</id>
```

Comment:
This profile will be omitted entirely when merging to main. It isn't needed there.